tests/rptest/services/redpanda.py

```diff
@@ -1850,32 +1876,39 @@ def pod_container_ready(pod_name: str):
         # Call to rebuild metadata for all cloud brokers
         self.rebuild_pods_classes()
 
+    def is_operator_v2_cluster(self):
+        return len(self.__kubectl.cmd(['get', 'redpanda', '-n=redpanda'])) > 0
```

Comment:
what's up with `self.kubectl.cmd` vs `self.__kubectl.cmd` ?

Reply:
Code added in this PR should only use `self.kubectl`, never `self.__kubectl`. Fixed in incoming commit.